packages/terra-clinical-item-collection/CHANGELOG.md

```diff
@@ -3,6 +3,18 @@ ChangeLog
 
 Unreleased
 ----------
+* See Release Notes for Migration Guide
+
+### Added
+* onSelect, children, and requiredElements props
+* Item Component to use as for Item Collection children
+
+### Removed
+* rows and onChange props
+
+### Changed
+* Item Collection layout is no longer rendered via the first row, but the requiredElements object
```

Comment:
There are several references to requiredElements that may not make contextual sense after flattening the props